website/source/docs/providers/aws/r/db_option_group.html.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: "aws"
+page_title: "AWS: aws_db_option_group"
+sidebar_current: "docs-aws-resource-db-option-group"
+---
+
+# aws\_db\_option\_group
+
+Provides an RDS DB option group resource.
+
+## Example Usage
+
+```
+resource "aws_db_option_group" "bar" {
+	option_group_name = "option-group-test-terraform"
+	option_group_description = "Terraform Option Group"
+	engine_name = "sqlserver-ee"
+	major_engine_version = "11.00"
+
+	option {
+		option_name = "mirroring"
+	}
+
+	option {
+		option_name = "TDE"
+	}
+	
+	apply_immediately = true
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `option_group_name` - (Required) The name of the Option group to be created.
+* `option_group_description` - (Required) The description of the option group.
+* `engine_name` - (Required) Specifies the name of the engine that this option group should be associated with..
+* `major_engine_version` - (Required) Specifies the major version of the engine that this option group should be associated with.
+* `parameter` - (Optional) A list of Options to apply.
```

Comment:
The example/code has these called `option` but here it's called `parameter`. Looks like `option` is the one that's closer to the API so this is the only place to get to match the rest I think.